lib/dialects/postgres/data-types.ts

```diff
@@ -0,0 +1,720 @@
+import wkx from 'wkx';
+import { addTicks, generateEnumName } from '../../utils';
+import type {
+  AcceptableTypeOf,
+  DataTypes,
+  RawTypeOf,
+  SetDialectNamesSymbol,
+  StringifyOptions,
+  DataType,
+  BindParamOptions,
+  DialectTypeMeta,
+} from '../abstract/data-types';
+import * as BaseTypes from '../abstract/data-types';
+import range from './range';
+
+const kSetDialectNames: SetDialectNamesSymbol = Symbol.for(
+  'sequelize.internal.data-types.set-dialect-names',
```

Comment:
> As I understand it this using the registry so third-party dialects can access the symbol?

More or less.  I intentionally wasn't exporting it since its not intended for user apis, but that'd be work too.